apps/labs/posts/sangho-internship-blogpost.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Sangho's Internship at Quansight with PyTorch-Ignite project"
+published: October 6, 2022
+author: sangho-lee
+description: 'Blogpost of working on the PyTorch-Ignite project during internship at Quansight'
+category: [Machine Learning, OSS Experience]
+featuredImage:
+  src: /posts/sangho-internship-blogpost/sangho1.png
+  alt: 'An image of the intern, Sangho.'
+hero:
+  imageSrc: /posts/sangho-internship-blogpost/ignite_logo_mixed.png
+  imageAlt: 'PyTorch-Ignite Logo'
+---
+
+Hey, I'm [Sangho Lee], a master's student from Seoul National University.
+I have participated to the [PyTorch-Ignite](https://github.com/pytorch/ignite) especially with distibuted computations at Quansight Labs.
+First thing I contributed was imprving test code of each metrics computated on DDP (Distributed Data Parallel) config.
+Second is modifying all_reduce and all_gather methods to take group arguments
+
+## About PyTorch-Ignite and distributed computations
+
+PyTorch-Ignite is a high-level library to help with training and evaluating neural networks in PyTorch flexibly and transparently.
+
+When you have a large batch of dataset such as training images, or videos then your device can’t afford more than a few datasets then it is natural to use multi-gpu in distributed configuration.
+
+In this case, data is transferred to multiple devices for computations then after calculation, it gathers outputs from each devices.
+<img alt="ddp configuration" src="/posts/sangho-blog-post/ddp1.png" />
```

Comment:
Our blog rendering likes it when you close html tags explicitly, i.e. with `/img>` rather than `/>`.
You might also consider using markdown for images if you aren't using any explicit html to place the image in a certain way on teh page. E.g. ![ddp configuration](/posts/sangho-blog-post/ddp1.png).